__tests__/app.test.js

```diff
@@ -156,5 +156,61 @@ describe("/api/articles", () => {
         const { message } = body;
         expect(message).toBe("Endpoint not found");
       });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200: Should return an array of comments for the given article ID with the required properties", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(5);
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
+        });
+      });
+  });
+  test("GET 200: Should return an array sorted by most recent comment first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSorted({ key: "created_at", descending: true });
+      });
+  });
+  test("GET 200: Should return an empty array if the article has no comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments.length).toBe(0);
+      });
+  });
```

Comment:
^^^ wonderful test, it shows you're really considering the boundaries of your endpoint and database! I think you should also prove it is an array here (.length exists on strings too)